tests/sentry/snuba/metrics/test_mqb_query_transformer.py

```diff
@@ -0,0 +1,707 @@
+                Condition(
+                    lhs=Column(name="timestamp", entity=None, subscriptable=None, key=None),
+                    op=Op.GTE,
+                    rhs=datetime.datetime(2022, 3, 24, 11, 11, 35, 447729, tzinfo=pytz.utc),
+                ),
+                Condition(
+                    lhs=Column(name="timestamp", entity=None, subscriptable=None, key=None),
+                    op=Op.LT,
+                    rhs=datetime.datetime(2022, 6, 22, 11, 11, 35, 447729, tzinfo=pytz.utc),
+                ),
+                Condition(
+                    lhs=Column(name="project_id", entity=None, subscriptable=None, key=None),
+                    op=Op.IN,
+                    rhs=[11],
+                ),
+                Condition(
+                    lhs=Column(name="org_id", entity=None, subscriptable=None, key=None),
+                    op=Op.EQ,
+                    rhs=11,
+                ),
+            ],
+            having=[],
+            orderby=[],
+            limitby=None,
+            limit=Limit(limit=51),
+            offset=Offset(offset=0),
+            granularity=None,
+            totals=None,
+        ),
+        MetricsQuery(
+            org_id=11,
+            project_ids=[11],
+            select=[
+                MetricField(
+                    op="count_web_vitals_measurements",
+                    metric_name="d:transactions/measurements.cls@millisecond",
+                    args=["good"],
+                    alias="count_web_vitals_measurements_cls_good",
+                ),
+                MetricField(
+                    op="count_web_vitals_measurements",
+                    metric_name="d:transactions/measurements.fid@millisecond",
+                    args=["meh"],
+                    alias="count_web_vitals_measurements_fid_meh",
+                ),
+                MetricField(
+                    op="count_web_vitals_measurements",
+                    metric_name="d:transactions/measurements.fp@millisecond",
+                    args=["good"],
+                    alias="count_web_vitals_measurements_fp_good",
+                ),
+                MetricField(
+                    op="count_web_vitals_measurements",
+                    metric_name="d:transactions/measurements.lcp@millisecond",
+                    args=["good"],
+                    alias="count_web_vitals_measurements_lcp_good",
+                ),
+                MetricField(
+                    op="count_web_vitals_measurements",
+                    metric_name="d:transactions/measurements.fcp@millisecond",
+                    args=["meh"],
+                    alias="count_web_vitals_measurements_fcp_meh",
+                ),
+            ],
+            start=datetime.datetime(2022, 3, 24, 11, 11, 35, 447729, tzinfo=pytz.utc),
+            end=datetime.datetime(2022, 6, 22, 11, 11, 35, 447729, tzinfo=pytz.utc),
+            granularity=Granularity(granularity=3600),
+            where=None,
```

Comment:
btw will there be a filter in the resulting SnQL on the metric_ids?

Reply:
Yes we do that and for org, project_id, start and end times as well